src/libmongoc/src/mongoc/mongoc-collection.c

```diff
@@ -166,7 +167,8 @@ _mongoc_collection_new (mongoc_client_t *client,
                         const char *collection,
                         const mongoc_read_prefs_t *read_prefs,
                         const mongoc_read_concern_t *read_concern,
-                        const mongoc_write_concern_t *write_concern)
+                        const mongoc_write_concern_t *write_concern,
+			const mongoc_timeout_t *timeout_ms)
```

Comment:
Don't forget to run `clang-format` before merging.

Reply:
👍